### PR TITLE
Move verifier package to its own submodule

### PR DESCRIPTION
**Pull request by alexmwu**
This should remove the large increase in dependencies due to #419. We temporarily include replace directives in cmd/go.mod and launcher/go.mod for now while we work on moving the package to a submodule. We also remove some unnecessary utilities like `FetchAttestation` and tests that won't work except on GCE.

# go-tpm-tools Breaking Changes
* Move package `verifier` from go-tpm-tools to a new submodule github.com/google/go-tpm-tools/verifier
  * `verifier.Client`, `verifier.Challenge`, etc.
* Move package `fake` from go-tpm-tools to a new submodule github.com/google/go-tpm-tools/verifier.
  * `fake.Claims`, `fake.NewClient`, etc.
* Move package `oci` and `cosign` from go-tpm-tools to a new submodule github.com/google/go-tpm-tools/verifier.
  * `oci.Signature`, `cosign.Sig`, etc.
* Move package `rest`  from go-tpm-tools to a new submodule github.com/google/go-tpm-tools/verifier.
  * `rest.NewClient`, `rest.BadRegionError`, etc.
* Move package `util` from go-tpm-tools to a new submodule github.com/google/go-tpm-tools/verifier.
  * `util.Instance`, `util.MetadataServer`, `util.NewMetadataServer`, etc.

### CODE REVIEW

**Review by jkl73 — APPROVED** [review]

File: launcher/auth.go [hunk 1]
```diff
@@ -39,6 +40,7 @@ func Resolver(token string) remotes.Resolver {
 		return "", "", nil
 	}
 	authOpts := []docker.AuthorizerOpt{docker.WithAuthCreds(credentials)}
+	//nolint:staticcheck
```
Comment (jkl73) [inline launcher/auth.go:43]: I can try to fix this later

Comment (alexmwu) [reply launcher/auth.go:43]: That'd be great, thanks!

**Pull request merged**